program/mobilenets-armcl-opencl/mobilenet.cpp

```diff
@@ -166,10 +177,11 @@ void setup_mobilenet(GraphObject& graph,
     graph << target_hint
           << get_convolution_method()
 #if defined(ARMCL_18_05_PLUS)
-          << DepthwiseConvolutionMethod::OPTIMIZED_3x3
+          << DepthwiseConvolutionMethod_OPTIMIZED_3x3
           << InputLayer(TensorDescriptor(input_shape, DATATYPE),
                 arm_compute::support::cpp14::make_unique<CKInputAccessor>(input_data_buffer))
 #else
+          // Can't specifiy depthwise convolution method for ArmCL before 18.05 
```

Comment:
// For ArmCL before 18.05, the optimized 3x3 depthwise convolution method is used by default.